internal/test/daemon/swarm.go

```diff
@@ -28,6 +28,18 @@ func (d *Daemon) StartAndSwarmInit(t testingT) {
 	d.SwarmInit(t, swarm.InitRequest{})
 }
 
+// StartAndSwarmInitWithOption starts the daemon (with busybox) and init the swarm with params
+func (d *Daemon) StartAndSwarmInitWithOption(t testingT, initReq swarm.InitRequest) {
```

Comment:
Same as above, we could update `StartAndSwarmInit` to take *operations` with one for `InitRequest`

```go
func (d *Daemon) StartAndSwarmInit(t testingT, ops ...func(*swarm.InitRequest{})) {
    initRequest := &swarm.InitRequest{}
    for _, op := range ops { op(initRequest) }
    // …
    d.SwarmInit(t, *initRequest)
}
```